lib/views/help/requesting.html.erb

```diff
@@ -201,7 +201,7 @@
         </li>
         <li>
           <p>
-            <strong>Don’t carpet-bomb:</strong>
+            <strong>Send targetted requests:</strong>
```

Comment:
typo:

```diff
- targetted
+ targeted
```

+1 on changing this; I've pointed to it a few times recently and grimaced a little when reading.

I wonder if this is a bit close to "Word your question carefully" and "Fine-tune your request".

This point is specifically trying to avoid bulk requesting in a way that's disruptive – the word scattergun comes to mind, but annoyingly similar connotations. Might be a nice alternative to [indiscriminate](https://www.wordhippo.com/what-is/another-word-for/indiscriminate.html)? If we can't think of anything else then happy with "Send targeted requests".

Reply:
thank you